tasks/lzards-backup/src/index.ts

```diff
@@ -0,0 +1,277 @@
+import AWS from 'aws-sdk';
+import got from 'got';
+import Logger from '@cumulus/logger';
+import { Context } from 'aws-lambda';
+
+import { CumulusMessage, CumulusRemoteMessage } from '@cumulus/types/message';
+import { getCollections } from '@cumulus/api-client/collections';
+import { getLaunchpadToken } from '@cumulus/launchpad-auth';
+import { getRequiredEnvVar } from '@cumulus/common/env';
+import { getSecretString } from '@cumulus/aws-client/SecretsManager';
+import { inTestMode } from '@cumulus/aws-client/test-utils';
+import { parseS3Uri } from '@cumulus/aws-client/S3';
+import { PartialCollectionRecord } from '@cumulus/types/api/collections';
+import { runCumulusTask, CumulusMessageWithAssignedPayload } from '@cumulus/cumulus-message-adapter-js';
+import { s3 as coreS3, sts } from '@cumulus/aws-client/services';
+
+import { ChecksumError, CollectionError, GetAuthTokenError } from './errors';
+import { isFulfilledPromise } from './typeGuards';
+import { makeBackupFileRequestResult, HandlerEvent, MessageGranule, MessageGranuleFilesObject } from './types';
+
+const log = new Logger({ sender: '@cumulus/lzards-backup' });
+
+const CREDS_EXPIRY_SECONDS = 3600;
```

Comment:
I'm not certain that's actually true because it doesn't give us an out other than a redeploy if something is wrong.     

This value is utterly arbitrary, and it's difficult to say what it should be as LZARDS time to backup isn't concrete/understood. 

I'll update this to be a TF variable for Core and set the default to something hopefully useful.

Reply:
Reminder to consider whether you want this to be a TF variable. I don't think it's necessary per se

Reply:
It isn't, but I think we do want it in there. 

e1ad17667 -- Updated to allow S3 link config, tightened up the sts creds to the lambda timeout time.